src/App.js

```diff
@@ -1,31 +1,60 @@
 import './App.css';
-import movieData from './movieData'
-import Theatre from './components/Theatre/Theatre'
+import movieData from './movieData';
+import Theatre from './components/Theatre/Theatre';
+import SinglePoster from './components/SinglePoster/SinglePoster';
+import ReturnButton from './components/ReturnButton/ReturnButton'
 import { Component } from 'react';
 
 class App extends Component {
   constructor () {
     super();
     this.state = {
-      movies: movieData.movies
+      movies: movieData.movies,
+      movieId: 0
     }
   }
 
+  posterClick = (id) => {
+    this.setState({ movieId: id })
+  }
+
+  deselect = () => {
+    this.setState({ movieId: 0 })
+  }
+
+  componentDidMount = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
```

Comment:
Check that Rubric - apiCalls should be broken out into their own file per React best practices